### PR TITLE
Feature/update seed phrase wording

### LINKED ISSUE #2175: Update 'seed phrase' wording to 'Secret Recovery phrase'
### Problem:
User's wallet safety depends on their ability to keep their seed phrase safe, secret, and retrievable if/when they need to recover their wallet. However, the term 'seed phrase' isn't descriptive on its own, and often people don't read through our seedphrase flow to fully grasp the importance of it. 

### Acceptance criteria:
To help improve the comprehension of why seed phrase is important and how to keep it safe, we want to
- Change the naming to 'Secret Recovery phrase'
- Add a quick video explainer as an additional & more engaging learning medium in the onboarding flow of the app (also available after the user has onboarded): 
- Don't let the video file make the app size (significantly) heavy such that it makes the app slow or run into storage limit issues.

Figma: https://www.figma.com/file/Di5k5DTvxDWGv5DtIhwMb8/secret-recovery-phrase?node-id=2%3A4739

Video: (with subtitles file) https://github.com/MetaMask/metamask-extension/tree/develop/app/images/videos/recovery-onboarding
https://drive.google.com/file/d/1wGnIPb7OJXB7UeXt7k52vJAI4WgSB3HV/view?usp=sharing

### Out of scope (will add to another analytic story)
Include metric event so we can measure well how many people are viewing the video via mobile app during onboarding, and from settings (and can view this new metric in relation with how many people are completing the backup flow (effectiveness of the video))

### PR DESCRIPTION
**Pull request by rickycodes**
**Description**

This PR updates seed phrase wording (to secret recovery phrase). It also includes a video re: seed phrase education.

<strike>opening as a draft until i get a format of the video that works with `react-native-video`</strike>

**Checklist**

* [x] There is a related GitHub issue
* [x] Tests are included if applicable
* [x] Any added code is fully documented

**Issue**

Resolves #2175

### CODE REVIEW

File: app/components/UI/SeedPhraseVideo/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		overflow: 'hidden',
+		width: '100%',
+		height: 180,
+		flex: 1
+	}
+});
+
+// eslint-disable-next-line import/no-commonjs
+const vid = require('../../../../app/videos/placeholder.mp4');
+const VTT_URI =
```
Comment (andrepimenta) [inline app/components/UI/SeedPhraseVideo/index.js:17]: I think this makes sense to be a local resource

**Review by andrepimenta — APPROVED** [review]
I think it would be better if 'https://github.com/MetaMask/metamask-extension/blob/develop/app/images/videos/recovery-onboarding/subtitles-en.vtt' was a local resource.
But other than that, looks good to me!

**Comment by rickycodes** [comment]
> I think it would be better if 'https://github.com/MetaMask/metamask-extension/blob/develop/app/images/videos/recovery-onboarding/subtitles-en.vtt' was a local resource.

I completely agree, but it doesn't seem possible... unfortunately.

**Comment by mobularay** [comment]
Video works fine, however @rickycodes and @mobularay have agreed to consider multilingual/English only subtitles as out-of-scope for this work, due to additional more work required to be done (see below), and was not included in this card/ticket. 

https://github.com/react-native-video/react-native-video/issues/1864
https://consensys.slack.com/archives/GTQAGKY5V/p1618515047087600?thread_ts=1618506468.069900&cid=GTQAGKY5V

**Comment by ibrahimtaveras00** [comment]
Hmmm, I'm wondering if this https://github.com/MetaMask/metamask-mobile/issues/2175#issuecomment-806276867 meant all places in app should be replaced @omnat?  If so I'm still seeing some areas where it isn't:

image

image

image

image

image

Don't worry, this is just a new test account with 0 funds
image

After successfully backing up:
image

**Comment by rickycodes** [comment]
@ibrahimtaveras00 those last few instances should be good now :)

**Review by ibrahimtaveras00 — APPROVED** [review]
Fixes look good, QA Passed 👍🏽

**Pull request merged**